podman/api_connection.py

```diff
@@ -90,14 +90,16 @@ def post(self, path, params=None, headers=None, encode=False):
             headers = {}
 
         if encode:
-            if "content-type" not in set(key.lower() for key in headers) and params:
+            if (
+                "content-type" not in set(key.lower() for key in headers)
+                and params
+            ):
```

Comment:
Is this an old formatting thing? I just ran black and I'm not getting this being updated. Also we switched to linelength 100 in #39 so this shouldn't be updated

Reply:
@mwhahaha I updated my preferences for formatting.